lib/adoptoposs_web/live/landing_page_live.ex

```diff
@@ -100,6 +101,12 @@ defmodule AdoptopossWeb.LandingPageLive do
     assign(socket, interests: Communication.list_user_interests(user))
   end
 
+  defp put_project_interests(socket, user) do
+    project_interests = Submissions.list_user_project_interests(user)
+
+    assign(socket, project_interests: project_interests)
+  end
```

Comment:
I think we would need to put the logged in navigation into a separate live component were the project interests are loaded load . Else `project_interests` would only be available for the landing page. Other pages would break (try navigating to the explore page and reload).

(With a component we can later also think about broadcasting incoming interests to make it update in realtime)

Reply:
You have the point. Let me address this idea.